ts/nni_manager/training_service/remote_machine/shellExecutor.ts

```diff
@@ -110,6 +109,15 @@ class ShellExecutor {
         this.sshClient.end();
     }
 
+    public async getIsWindows(): Promise<boolean>{
+        const result = await this.execute("ver")
```

Comment:
We have `ShellExecutor.isWindows` to check if is windows, if necessary, change `isWindows` to `public`.

Reply:
Done.